protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "Manual_ControlProto";
+
+service ManualControlService
+{
+
+    
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc SetPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
+    
+    
+    rpc SetAltitudeControl (AltitudeControlRequest)  returns(AltitudeControlResponse) {}
+    
+      /*
+     * Send command to command to the drone to move forwards or backwards 
+     */
+    rpc Axismotion (MotionRequest) returns (MotionResponse) {}
+
+     
+
+
+
+}
+
+
+message PositionControlRequest{}
+message PositionControlResponse{
+    ManualControlResult manual_control_result = 1;
+}
+
+message AltitudeControlRequest{}
+message AltitudeControlResponse{
+    ManualControlResult manual_control_result = 1;
+}
+
+
+
+message MotionRequest {
+    sint32 xmotion = 1 ;                    // value between -1000 to 1000. negative-> backwards, positive-> forwards 
```

Comment:
```suggestion
    float32 x = 1 ;                    // value from -1 to 1. negative-> backwards, positive-> forwards 
```

I wouldn't constrain it to a resolution of 1000 but just use float.

Reply:
I did it base on the messages documentation which varies betweem -1000 to 1000

Reply:
Yes that's a good point.

@julianoes: I'm not completely sure I understand why a float is better, given that the mavlink message has a fixed resolution. My concern is that I could imagine ending up in situations where the user sends a float, and receives back another float but both were represented by the same MAVLink message.

Reply:
I see. I generally advocate not to be guided by the MAVLink API but try to do what is right, makes sense. And it doesn't really make sense to put values from -1000 to 1000 into a type which allows for -32767 to 32768.

If we really do make it from -1000 to 1000 we should use the same type as the MAVLink message as well which is `int16`.

> @JonasVautherin
> and receives back another float but both were represented by the same MAVLink message

How would that happen exactly?

Reply:
I am not sure that int16 exis on the proto library that is the reason I  use sint32